CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 0.73.0 / 2024-04-03
+
+* [CHANGE/BUGFIX] Add `proxyURL` validation for ServiceMonitor, PodMonitor and Probe objects. #6464
+  *It could be a breaking change for those who rely on the configuration reloader sidecar to expand environment variables in the generated configuration file.*
```

Comment:
```suggestion
* [CHANGE/BUGFIX] Add `proxyURL` validation for ServiceMonitor, PodMonitor and Probe objects. **It could be a breaking change for those who rely on the configuration reloader sidecar to expand environment variables in the generated configuration file.** #6464  
```

Reply:
done